bibs/Glarus.json

```diff
@@ -11,7 +11,24 @@
     "country": "Schweiz",
     "data": {
         "baseurl": "https://landesbibliothek.gl.ch",
-        "opacdir": "opax"
+        "mediatypes": {
+            "0": "ALL",
+            "1": "BOOK",
+            "10": "MAP",
+            "11": "DVD",
+            "2": "MAGAZINE",
+            "3": "CD_SOFTWARE",
+            "34": "AUDIOBOOK",
+            "5": "AUDIO_CASSETTE",
+            "6": "CD_MUSIC"
+        },
+        "opacdir": "opax",
+        "searchtable": [
+            2,
+            3,
+            4,
+            5
+        ]
```

Comment:
`searchtable` lists the column indices that should be parsed for details about the item, so the columns that contain title, author, year, category, etc. I think these are only columns `2, 3, 4` in this case.

Reply:
You are right!